sass/cd/cd-header/_cd-dropdowns.scss

```diff
@@ -19,6 +19,20 @@
   }
 }
 
+svg.arrow-down {
+  width: 12px;
+  height: 12px;
+  fill: $cd-white;
+}
+
+.open {
+  & > [data-toggle="dropdown"] {
+    & > svg.arrow-down {
```

Comment:
why the nesting here? no need for `&` when the character that follows is a space. I understand the `.open` block in case we wanted to apply other changes but this would work and is easier for me to read:

```scss
.open {
  [data-toggle="dropdown"] > svg.arrow-down {
    transform: rotate(180deg);
  }
}
```

Reply:
Cool. This is legacy stuff. I will simplify for readibility